test/Microsoft.Azure.SignalR.Tests/TestServiceConnectionContainer.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.SignalR.Protocol;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Microsoft.Azure.SignalR.Tests
+{
+    internal sealed class TestServiceConnectionContainer : ServiceConnectionContainerBase
```

Comment:
We have `TestServiceConnectionContainer` in test common project, can that be leveraged?

Reply:
I'm afraid they are for different scenarios.